### PR TITLE
App builder 7.3 | User Guide | Take 4 | LRDOCS-7120

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-7120

### CODE REVIEW

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md [hunk 1]
```diff
@@ -2,9 +2,24 @@
...
+App Builder is a low-code application development solution for Liferay DXP. Build a data model, define the application's forms (for entering data) and tables (for viewing and managing data records), then deploy your application to your Liferay DXP site, or as a completely standalone application. With App Builder, creating functional business applications with dedicated user interfaces and data separation is accomplished without a single line of code.
 
-<!-- Wiz-bang screenshot of App Builder -->
+![Two separate applications are deployed for this App Builder object.](./app-builder-overview/images/01.png)
+
+Why use app Builder when the Liferay Forms application includes almost identical form-building capability? If you're asking that question, you may not need the added features and flexibility App Builder provides. Here's an overview of the differences:
+
+| Capability | Liferay Forms | App Builder | 
+|------------|:-------------:|:-----------:|
```
Comment (rbohl) [inline docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md]: FYI, the 
|:------:| 
stuff makes the cell content centered, which I thought looked better. Sorry if the left justified style was intentional.

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md [hunk 2]
```diff
@@ -42,20 +57,20 @@ Once an object is defined and at least one form and table view are created, you
 **Widget:** Deploy the application to the widget menu, and add it to a page using the Add Widgets menu (from Add &rarr; Widgets, search for your application by name). Deploying as a widget really deploys three variations of the application to the Add Widgets menu, each with a different label. Each option exposes different features for your application's users:
 
 | Widget Menu Label     | Submit Form Entries | View Entries | Update and Delete Entries |
-|-----------------------|-----------|------------|----------------|
+|-----------------------|:-----------:|:------------:|:----------------:|
```
Comment (rbohl) [inline docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md]: centered again

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-an-application-with-app-builder.md [hunk 3]
```diff
@@ -94,17 +92,39 @@ We will deploy two applications for the Time Off Request object: one is a widget
...
+You can undeploy and redeploy App Builder applications at any time (e.g., if a new set of time off management applications are created). If you undeploy an application, any entries already submitted are preserved, but no new entries can be added. To undeploy,
 
 1. Open the custom object (Control Panel &rarr; App Builder &rarr; Custom Objects).
 
 1. From the _Apps_ tab, click the kebab menu for the app you want to undeploy.
 
 1. Select _Undeploy_.
 
-<!-- Potentially obvious but, does Undeploying an application pose any risk of losing entries that have been submitted? What if I change the data object after entries have been submitted? -->
+## Editing an Object with Data Records
```
Comment (rbohl) [inline docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-an-application-with-app-builder.md]: New section based on your comment

**Comment by jrhoun** [comment]
Merged. See https://github.com/jrhoun/liferay-learn/commit/2af7c71c09b6450d2c72db6035723252edbbaf5c

One issue that I think we should circle back to is that some (most?) of the screenshots that were included seemed a little blurry/low quality to me. You may want to consider doubling back to update those in the future.

**Comment by jrhoun** [comment]
Thanks @rbohl !

**Pull request closed without merging**